docs/TektonConfig.md

```diff
@@ -248,6 +248,7 @@ Example:
 chain:
   disabled: false                  # - `disabled` : if the value set as `true`, chains feature will be disabled (default: `false`)
   targetNamespace: tekton-pipelines
+  generateSigningSecret: true # default value: false
```

Comment:
After setting this value, signing-secrets were not updated

Reply:
You need to delete the chains secret installerset. This is the way the operator handles the objects that are generated by the installersets 
@piyush-garg can you confirm please, that in order to update an object after a config change, we have to delete the installerset, in order to the reconciler to restart it is creation operation

Reply:
I think reconciler should take care of that and user shouldn't have to go and delete the installerset